lib/sidekiq-status.rb

```diff
@@ -57,7 +57,7 @@ def total(job_id)
     end
 
     def pct_complete(job_id)
-      ((at(job_id) / total(job_id).to_f) * 100 ).to_i if total(job_id).to_f > 0
+      get(job_id, :pct_complete).to_i
```

Comment:
I would refactor this so the only redis call you make is to retrieve the complete hash

```ruby
def pct_complete(job_id)
  get_all(job_id).fetch('pct_complete', 0)
end

def get_all(job_id)
  read_hash_for_id(job_id)
end
```